Meadowlark-js/packages/meadowlark-authz-server/src/security/JwtValidator.ts

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I'm not sure if I agree about moving this file.

What happens when we connect Meadowlark to an external OAuth2 system, rather than managing credentials directly in Meadowlark? Is this package (`meadowlark-authz-server`) intended to cover that situation?

Reply:
Not moved, just duplicated error responses. Actual JWT verification is still in meadowlark-core.